debian/patches/connectionmanager.js.patch

```diff
@@ -0,0 +1,1609 @@
+--- a/MediaBrowser.WebDashboard/dashboard-ui/bower_components/emby-apiclient/connectionmanager.js
```

Comment:
Because this file is minified, I chose to keep it as a full-replacement instead of a patch. Your patch seems to include the entire file which defeats the purpose of a patch.

I thought about prettyfying it, then patching it, but haven't have prioritzed researching that.

Unless you have an easier way to maintain modification of this file, let's leave it as a patch.

Reply:
I agree it's completely gross, but I have yet to find a mechanism for a Debian patch to *replace* a file rather than patch it. This hack was my compromise to get it to build.